### ISSUE TITLE
Please fix warnings

### ISSUE THREAD

**Issue by algorithmx** [JuliaPy/Pandas.jl#28]
### versioninfo
````
julia> versioninfo()
Julia Version 0.7.0-beta2.0
Commit b145832402* (2018-07-13 19:54 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2609 v4 @ 1.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, broadwell)
Environment:
  JULIA_DEPOT_PATH = /home/XXXXX/.julia7/

````
## WARNINGS

````
julia> using Pandas
[ Info: Precompiling module Pandas
WARNING: importing deprecated binding Base.mean into Pandas.
WARNING: importing deprecated binding Base.std into Pandas.
WARNING: importing deprecated binding Base.var into Pandas.
WARNING: importing deprecated binding Base.cov into Pandas.
WARNING: could not import Base.cummax into Pandas
WARNING: could not import Base.cummin into Pandas
WARNING: importing deprecated binding Base.median into Pandas.
WARNING: importing deprecated binding Base.quantile into Pandas.
WARNING: importing deprecated binding Base.rank into Pandas.
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:203
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.cov is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.mean is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.median is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.quantile is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.rank is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.std is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Base.var is deprecated: it has been moved to the standard library package `Statistics`.
Add `using Statistics` to your imports.
  likely near /home/XXXXX/.julia7/packages/Pandas/jZPC/src/Pandas.jl:208
WARNING: Broadcast.broadcast_indices is deprecated, use broadcast_axes instead.
  likely near /home/XXXXX/.julia7/packages/Nullables/IeVZ/src/nullable.jl:414

````

**Comment by malmaud** [comment]
I'll update for 1.0 soon.

**Comment by malmaud** [comment]
Fixed in c5320af4da28972e154805729806c38344ee57e7
